src/main/java/com/yana/internship/entity/Tariff.java

```diff
@@ -0,0 +1,30 @@
+package com.yana.internship.entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "tariff")
+public class Tariff {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
```

Comment:
Let's use Long for all ID's as we decided.

Reply:
Done